src/app/projects/projects.service.ts

```diff
@@ -64,16 +66,17 @@ export class ProjectsService {
           obj.content.supplementary_links || []
         ),
         publications: obj.publicationsInfo ?? [],
-        authors: obj.content.contributors?.map((author) => {
-          const names = author.name.split(',');
-          const formattedName = `${
-            names[names.length - 1]
-          } ${names[0][0].toUpperCase()}`;
-          return {
-            fullName: author.name,
-            formattedName,
-          };
-        }) || [],
+        authors: obj.content.contributors?.filter(c => !this.isWrangler(c))
+          .map((author) => {
+            const names = author.name.split(',');
```

Comment:
Would this name formatting fit better as a pipe?

Reply:
Good idea!

Reply:
Actually, while this is a good idea I think we should leave it for now. We are getting nice error reporting from this because the author formatting is failing for some projects due to malformed contributors. The try/catch block is catching this and logging the uuid causing the problem. This would be possible to achieve with a custom `ErrorHandler` but more difficult

Reply:
Are we using the Ui/ to clean our data? :)
We can run the same code on the values in the db and fix, can't we?
Regardless of using a pip, I think we should extract a function of this naming normalisation function.

Reply:
The DB can change regularly with updates to projects and/or additions to the catalogue. There's no way to ensure we don't change records that have a bad effect on the catalogue. I think this is a simple and effective way of achieving that. It just logs the project that causes the problem and continues loading the rest of the catalogue